Chummer/frmNotes.cs

```diff
@@ -92,6 +94,34 @@ private void frmNotes_Shown(object sender, EventArgs e)
             txtNotes.SelectionLength = 0;
             txtNotes.SelectionStart = txtNotes.TextLength;
         }
+        private void btnOK_Click(object sender, EventArgs e)
+        {
+            _strNotes = txtNotes.Text;
+            DialogResult = DialogResult.OK;
+        }
+
+        private void btnCancel_Click(object sender, EventArgs e)
+        {
+            DialogResult = DialogResult.Cancel;
+        }
+
+        private void btnColorSelect_Click(object sender, EventArgs e)
+        {
+            //colorDialog1.Color = ColorManager.GenerateCurrentModeColor(_colNotes);
+            _colNotes = colorDialog1.Color; //Selected color is always how it is shown in light mode, use the stored one for it.
+            var resNewColor = colorDialog1.ShowDialog();
+            if (resNewColor == DialogResult.OK)
+            {
+                //_colNotes = ColorManager.GenerateModeIndependentColor(colorDialog1.Color);
+                _colNotes = colorDialog1.Color; //Selected color is always how it is shown in light mode.
+                updateColorRepresentation();
+            }
+        }
+        private void txtNotes_TextChanged(object sender, EventArgs e)
+        {
+            btnColorSelect.Enabled = txtNotes.Text.Length > 0;
```

Comment:
I think there's a separate property that lets you get a text length directly? I'm not sure though. You can also check to see if the text is empty.

Reply:
Yes, that's `txtNotes.TextLength`. Will switch to that one with the next revision.